includes/admin/upgrades/upgrade-functions.php

```diff
@@ -892,3 +897,31 @@ function give_v187_upgrades(){
 		}
 	}
 }
+
+/**
+ * Update Capabilities for Give_Worker User Role.
+ *
+ * This upgrade routine will update access rights for Give_Worker User Role.
+ *
+ * @since      1.8.8
+ */
+function give_v188_upgrades() {
```

Comment:
@ravinderk I've checked the code again and tried without the upgrade script for capabilities. But, it is not affecting to existing users without an upgrade. If I run an upgrade, it is immediately adding capabilities to existing users with `give_worker` role